src/components/Menu/LanguageDropdownSelector/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Select from 'react-select';
+import { optionStyles } from '../ReactSelectStyles/option-styles';
+import { controlStyles } from '../ReactSelectStyles/control-styles';
+import { dropdownIndicatorStyles } from '../ReactSelectStyles/dropdown-indicator-styles';
+import { groupHeadingStyles } from '../ReactSelectStyles/group-heading-styles';
+import { noIndicatorSeparator } from '../ReactSelectCustomComponents/no-indicator-separator';
+import { FormatOptionLabelWithLanguageLogo } from '../ReactSelectCustomComponents/Formatters/FormatOptionLabelWithLanguageLogo';
+import { menuListStyles } from '../ReactSelectStyles/menu-list-styles';
+import languageLabels from '../../../maps/language';
+import { ReactSelectOption } from '../react-select-option-types';
+import { PREFERRED_LANGUAGE_KEY } from '../../../utilities/language/constants';
+import { createLanguageHrefFromDefaults, getLanguageDefaults } from '../../common/language-defaults';
+import { safeWindow } from '../../../utilities/browser/safe-window';
+import { navigate } from 'gatsby';
+import { DEFAULT_LANGUAGE } from '../../../../data/createPages/constants';
+
+export const LanguageDropdownSelector = ({
+  language,
+  languages,
+  showDefaultLink,
```

Comment:
This is never used. Also, shouldn't this be in a LanguageDropdownSelector.tsx file and the index.ts should just be a series of exports